src/main/steps/applicant1/dividing-money-property/content.ts

```diff
@@ -1,46 +1,94 @@
+    isDivorce ? 'for divorce' : 'to end your civil partnership'
+  }.`,
+  noSelectedWarning: `It's important to have a financial order in place by the time your ${
+    isDivorce ? 'divorce is finalised' : 'civil partnership is legally ended'
+  }.
+  Otherwise you or your ${partner} could make claims on each other's finances, after the ${
+    isDivorce ? 'marriage' : 'civil partnership'
+  } is ended.
+  Even if you have kept your finances separate during the ${
+    isDivorce ? 'marriage' : 'civil partnership'
+  }. If you select yes, then you do not have to go ahead with the application or pay any additional fees. It just gives you the option to apply later in the process, should you want&nbsp;to.`,
+  doYouWantToApplyForFinancialOrder: 'Do you want to apply for a financial order?',
+  yes: 'Yes, I want to apply for a financial order',
+  no: 'No, I do not want to apply for a financial order',
 });
 
-const cy: typeof en = ({ partner }: CommonContent) => ({
-  title: "Rhannu eich arian a'ch eiddo",
-  line1: `Gan amlaf, mae'n symlach ac yn llai costus os ydych yn cytuno â'ch ${partner} ynghylch sut i rannu eich cynilion, eiddo, pensiynau ac asedau eraill. Mae yna wasanaethau cyfryngu ar gael i'ch helpu i ddod i gytundeb. Byddwch yn cael dolenni i ragor o wybodaeth ar ôl i chi gyflwyno'r cais hwn.`,
-  agreeOnDividingAssets: 'Os ydych yn cytuno ynghylch rhannu arian ac eiddo',
-  agreeOnDividingAssetsDetails: `Gallwch ofyn i'r llys wneud eich cytundeb yn gyfreithiol rwymol. Gelwir hyn yn gwneud cais am 'orchymyn cydsynio' (sef math o orchymyn ariannol). Mae ffi ychwanegol o ${getFee(
-    config.get('fees.consentOrder')
-  )}. Gallwch gael cyngor cyfreithiol neu ofyn i gyfreithiwr ddrafftio gorchymyn cydsynio ar eich rhan. Byddwch yn cael dolenni i arweiniad pellach ar ôl i chi gyflwyno'r cais hwn.`,
-  disagreeOnDividingAssets: 'Os ydych yn anghytuno ynghylch rhannu arian ac eiddo',
-  disagreeOnDividingAssetsDetails:
-    "Gallwch ofyn i'r llys benderfynu ar eich rhan. Gelwir hyn yn gofyn i'r llys wneud 'gorchymyn ariannol'. Mae hyn yn golygu y bydd y llys yn penderfynu sut y caiff asedau eu rhannu. Gallwch hefyd wneud cais am orchymyn ariannol i'ch plant, os yw'n briodol. Gall y llys hefyd orchymyn i daliadau cynhaliaeth gael eu gwneud.",
-  readMore: 'Darllenwch fwy am gynhaliaeth plant',
-  readMoreContent: `Dim ond o dan rai amgylchiadau y gall y llys wneud gorchmynion ariannol i blant. Gallwch ddod i gytundeb â'ch ${partner} a'i wneud yn gyfreithiol rwymol gyda gorchymyn cydsynio, neu gallwch ddefnyddio'r Gwasanaeth Cynhaliaeth Plant.`,
-  costs: `Rhaid i chi wneud cais ar wahân i'r llys am orchymyn ariannol, gan ddefnyddio ffurflen arall. Mae'n costio ${getFee(
-    config.get('fees.financialOrder')
-  )} yn ychwanegol.`,
+const cy: typeof en = ({ partner, isDivorce }: CommonContent) => ({
```

Comment:
Do we need the welsh translation on this before it goes in?

Reply:
@DelythJustice Welsh translations are usually handled in their own ticket.

Reply:
Yeah we had a chat about the Welsh in one of our team meetings and Kate seemed to imply we could merge with just the English to start with. I’ll highlight that the Welsh is missing again with a comment in the ticket when it goes to awaiting sign-off